formulas/github-get-archive/functions/github-get-archive

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Path to Github, Inc. repository.
+#
+#$1              - Path to Github repository.
```

Comment:
I followed your comment too litterally  "No spaces at the beginning, a single hyphen separating the option from the description."